src/main/java/datawave/query/util/TypeMetadata.java

```diff
@@ -103,6 +104,48 @@ public Collection<String> getTypeMetadata(String fieldName, String ingestType) {
         return Sets.newHashSet(map.get(fieldName));
     }
     
+    /**
+     * Returns a set of all Normalizer names associated with the given fieldName. This is similar to calling .fold().get(fieldName)
+     *
+     * @param fieldName
+     *            a field name against which to search for any associated Normalizer Types
+     * @return a set of strings of associated Normalizer Types
+     */
+    public Set<String> getNormalizerNamesForField(String fieldName) {
+        if (fieldName == null || fieldName.isEmpty()) {
+            return Collections.emptySet();
+        }
+        
+        Set<String> normalizers = new HashSet<>();
+        for (Multimap<String,String> entry : this.typeMetadata.values()) {
+            normalizers.add(String.valueOf(entry.get(fieldName)));
```

Comment:
Why String.valueOf?  The entry.get(fieldName) is already a string.

Reply:
Good catch - changing to `.addAll()`